QtAwesome/iconic_font.py

```diff
@@ -10,12 +10,20 @@
 _default_options = {
     'color': QColor(50, 50, 50),
     'scale_factor': 0.9,
+    'rotation': False,      # Valid options are: 'spin', or 'pulse'
+    'parent': False,        # Ref to instance of widget holding the icon
+    'spin_interval': 10,    # miliseconds
+    'spin_step': 1,         # degrees
+    'pulse_interval': 300,  # miliseconds
+    'pulse_step': 45,       # degrees
```

Comment:
Well, styling properties belong more naturally to the painter than animations.
I guess that if someone wants to change styling behavior, the right thing to do is to use a custom painter. Hence I would not move the offset or rotation. I have some changes regarding offset, opacity, I just need to push.